bin/citrea/tests/bitcoin_e2e/prover_test.rs

```diff
@@ -307,14 +314,101 @@ impl TestCase for SkipPreprovenCommitmentsTest {
     }
 }
 
-#[tokio::test]
-async fn basic_prover_test() -> Result<()> {
-    TestCaseRunner::new(BasicProverTest).run().await
-}
-
 #[tokio::test]
 async fn prover_skips_preproven_commitments_test() -> Result<()> {
     TestCaseRunner::new(SkipPreprovenCommitmentsTest::default())
         .run()
         .await
 }
+
+struct LocalProvingTest;
+
+#[async_trait]
+impl TestCase for LocalProvingTest {
+    fn test_config() -> TestCaseConfig {
+        TestCaseConfig {
+            with_sequencer: true,
+            with_batch_prover: true,
+            with_full_node: true,
+            ..Default::default()
+        }
+    }
+
+    fn test_env() -> TestCaseEnv {
+        TestCaseEnv {
+            test: vec![
+                ("SHORT_PREFIX", "1"),
+                ("BONSAI_API_URL", ""),
+                ("BONSAI_API_KEY", ""),
+            ],
+            ..Default::default()
+        }
+    }
+
+    fn batch_prover_config() -> BatchProverConfig {
+        BatchProverConfig {
+            proving_mode: ProverGuestRunConfig::Prove,
```

Comment:
very very against proving on CI

Reply:
agreed 100%, that's why I ignored. but still didn't wanna delete an existing test case in case we make it work with cuda and at least it is somewhat feasible to wait.

Reply:
didnt realize it was ignored